### PR TITLE
Fix upload shapefile and gtfs

### PR DESCRIPTION
**Pull request by landonreed**
Fixes:

1. uploading of route shapefile for visual aid in the Editor #20
2. upload of GTFS in the FeedSourceTable or ProjectView #25

### CODE REVIEW

File: lib/editor/components/FeedInfoPanel.js [hunk 1]
```diff
@@ -64,16 +65,17 @@ export default class FeedInfoPanel extends Component {
   _openSnapshotModal = () => this.refs.snapshotModal.open()
 
   showUploadFileModal = () => {
+    const {displayRoutesShapefile, feedSource} = this.props
     this.refs.selectFileModal.open({
       title: 'Upload route shapefile',
-      body: 'Select a zipped shapefile to display on map:',
+      body: `Select a zipped shapefile to display on map. Note: this is only for use as a visual aid.`,
```
Comment (evansiroky) [inline lib/editor/components/FeedInfoPanel.js]: Should this be a message instead?

File: lib/manager/components/FeedSourceDropdown.js [hunk 2]
```diff
@@ -65,6 +65,7 @@ export default class FeedSourceDropdown extends Component {
   _onDelete = () => this.props.deleteFeedSource(this.props.feedSource)
 
   render () {
+    console.log(this.props)
```
Comment (evansiroky) [inline lib/manager/components/FeedSourceDropdown.js]: Please remove

File: lib/manager/containers/ActiveProjectViewer.js [hunk 3]
```diff
@@ -62,29 +62,29 @@ const mapDispatchToProps = (dispatch, ownProps) => {
...
+    onNewFeedSourceClick: () => dispatch(createFeedSource(projectId)),
+    saveFeedSource: (name) => dispatch(saveFeedSource({ projectId, name })),
+    searchTextChanged: (text) => dispatch(setVisibilitySearchText(text)),
+    thirdPartySync: (type) => dispatch(thirdPartySync(projectId, type)),
+    updateFeedSourceProperty: (feedSource, propName, newValue) => dispatch(updateFeedSource(feedSource, { [propName]: newValue })),
+    updateAllFeeds: (project) => dispatch(fetchFeedsForProject(project)),
+    updateDeployment: (deployment, changes) => dispatch(updateDeployment(deployment, changes)),
+    updateProjectSettings: (project, newSettings) => dispatch(updateProject(project, newSettings, true)),
+    uploadFeed: (feedSource, file) => dispatch(uploadFeed(feedSource, file)),
+    visibilityFilterChanged: (filter) => dispatch(setVisibilityFilter(filter))
```
Comment (evansiroky) [inline lib/manager/containers/ActiveProjectViewer.js]: At some point, we should refactor all of the mapDispatchToProps actions.  This is a lot of unneeded boilerplate.

File: lib/editor/components/FeedInfoPanel.js (hunk 1 above)
Comment (landonreed) [reply lib/editor/components/FeedInfoPanel.js]: Yes.

File: lib/manager/containers/ActiveProjectViewer.js (hunk 3 above)
Comment (landonreed) [reply lib/manager/containers/ActiveProjectViewer.js]: Yes, pretty much the only thing keeping that from happening is the `onComponentMount`, which has some custom logic that depends on dispatching actions.  But maybe this needs to be defined as its own action (e.g., `onActiveProjectViewerMount`).  Creating those can be a new PR.

**Pull request merged**